submissions/jeffersonMormah/domManipulation/contact.js

```diff
@@ -0,0 +1,45 @@
+let formContact = document.getElementById("contact-page");
+let fullName = document.getElementById("full-name");
+let email = document.getElementById("email");
+let contactSubject = document.getElementById("subject");
+let contactMessage = document.getElementById("message");
+let validName = /^[A-Za-z]/;
+let validEmail = /^+([]?+)*@+([-]?+)*({2,6})+$/;
+let validSubject = /^[A-Za-z]/;
+let validMessage = /^[A-Za-z]/;
+let nameField = "Please enter Your Name";
+let nameError = "It must contain only letters";
+let emailField = "Please enter your email address";
+let emailError = "Please enter a valid email address";
+let subjectField = "Please Enter a subject";
+let subjectError = "Enter selected subject";
+let messageField = "Please enter a Message";
+let messageError = "Please enter a Message";
```

Comment:
`messageError` and `messageField` have the same value. It better to just use one of them